src/domains/adapter/dto/index.ts

```diff
@@ -4,7 +4,6 @@ export interface AdaptProjectDataInputDTO {
   providerProjectId: string;
```

Comment:
Feel free to erase the 'DTO' part in the interface name if it is not being sent, recieved or returned by Database.